src/sentry/tasks/relocation.py

```diff
@@ -1103,6 +1092,9 @@ def notifying_users(uuid: str) -> None:
             hash = lost_password_hash_service.get_or_create(user_id=user.id).hash
             LostPasswordHash.send_relocate_account_email(user, hash, relocation.want_org_slugs)
 
+        relocation.latest_unclaimed_emails_sent_at = datetime.now()
```

Comment:
do we also want to set `latest_notified` here?

Reply:
No. `latest_notified` refers to the linear emails sent to to the owner to update them on whether or not their relocation started/succeeded/failed. This field refers to a single "claim your account" email blast to all unclaimed users. We may want to do more blasts after the relocation has landed, so we store the date here.